--- mod_websocket.lua	2021-02-10 15:09:44.000000000 +0000
+++ mod_websocket.lua	2021-02-10 15:10:15.000000000 +0000
@@ -304,6 +304,7 @@
 	response.headers.connection = "Upgrade";
 	response.headers.sec_webSocket_accept = base64(sha1(request.headers.sec_websocket_key .. "258EAFA5-E914-47DA-95CA-C5AB0DC85B11"));
 	response.headers.sec_webSocket_protocol = "xmpp";
+	module:fire_event("websocket-session", { session = session, request = request });

 	session.log("debug", "Sending WebSocket handshake");